ang/civicase/dashboard/directives/dashboard.directive.js

```diff
@@ -46,6 +46,7 @@
     function bindRouteParamsToScope () {
       $scope.$bindToRoute({ param: 'dtab', expr: 'activeTab', format: 'int', default: 0 });
       $scope.$bindToRoute({ param: 'drel', expr: 'filters.caseRelationshipType', format: 'raw', default: 'is_involved' });
+      $scope.$bindToRoute({ param: 'case_type_category', expr: 'filters.caseTypeCategory', format: 'raw', default: false });
```

Comment:
why not bind directly to `$scope.activityFilters.case_filter['case_type_id.case_type_category']`?

Reply:
if we bind to that expression directly there won't be a need for the `caseTypeCategoryWatcher` function.